### DISCUSSION TITLE
Virustotal detection

### DISCUSSION THREAD

**Discussion by ghost** [Kron4ek/minecraft-vortex-launcher#85] [General]
I downloaded a file from the latest release and thought to scan the file. After scanning, 15 vulnerabilities were found (https://www.virustotal.com/gui/file/461b8dc55a0923ec0ada54e19322d4b68ba21b067d2f067cd5dcd85a71f52f9c)

**Comment by Kron4ek** [comment]
https://github.com/Kron4ek/minecraft-vortex-launcher/issues/41#issuecomment-1033604934
